README.md

```diff
@@ -89,11 +83,10 @@ browse_metadata()
 Upon success, you should see:
 
 ```
-ℹ Three outputs have been saved to your output directory.
-ℹ Open the two HTML files in your browser for full-screen viewing.
+ℹ Three outputs have been saved to your output directory, and two outputs should have opened in your browser.
 ```
 
-The output files are saved to your working directory. You can change the save location by adjusting the `output_dir` argument. Examples of outputs are available in [inst/outputs](https://github.com/aim-rsf/browseMetadata/tree/main/inst/outputs).
```

Comment:
this could still mention the default is the current directory?

I don't think you need to refer to usethis, but maybe the text could say the location is "in your project" where project links to https://www.tidyverse.org/blog/2017/12/workflow-vs-script/, depending on who you think the target audience is.

Reply:
Do you mean something like this 

 "The output files are saved your project directory. You can change the save location by adjusting the `output_dir` argument. Examples ..."

Reply:
yes!

Reply:
Addressed here - https://github.com/aim-rsf/browseMetadata/pull/167/commits/dbe6e2045be0053f48fcc16875a0a88ca8487bb3